### PR TITLE
Prepared inputs for cmet and hif2a merck datasets

### PR DESCRIPTION
**Pull request by MCompChem**
## Input submission checklist

* [ x] created a directory for each system based on the [template directory][templates]
* [ x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [ x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [ x] added a PDB file with the protein named `protein.pdb`
* [x ] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x ] kept cystallographic waters and metals in the PDB file
* [x ] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [ x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Summary of changes
Reprepared protein structures with protein preparation in Maestro. Removed flip binding mode ligands for Hif2a dataset.

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

**Comment by IAlibay** [comment]
Thanks @MCompChem ! We will shortly review this.

**Comment by hannahbaumann** [comment]
Thank you @MCompChem! This generally looks great, just one thing we noticed is that some of the protonation states (e.g. ASP1222 in cmet) are different from the original structure prepped by Ross et al. 
Is there a way to prep the protein without changing the protonation states (and also not changing the histidine tautomers) in maestro?

**Comment by MCompChem** [comment]
It is quite interesting that in the prepared structure from Ross et al. this ASP1222 is not charged but this is probably a manual intervention that was not discussed in the paper to account for the neighboring ASP1204. 
I have rerun the preparation to add the caps without changing the already placed hydrogens so this should keep the assignments in the same way as in the publication.

Not sure what would be the best way forward scientifically. Ideally I would run the system twice with the different inputs.

**Comment by jameseastwood** [comment]
Consider for discussion in 2 July governance meeting

File: industry_benchmarks/input_structures/prepared_structures/merck/cmet/PREPARATION_DETAILS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Protein structure from Schrodinger public benchmark set was reprepared with Protein preparation wizard with capped termini (replace hydrogens was switched off to fix protonation states and tautomers for histidines, optimize hbond assignments was turned off and all other settings kept at default).
```
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/merck/cmet/PREPARATION_DETAILS.md:1]: Are there any Maestro version numbers we can point to?

**Review by IAlibay — CHANGES_REQUESTED** [review]
Thanks @MCompChem. There's just a few small things and then I think it's good to go!

File: industry_benchmarks/input_structures/prepared_structures/merck/cmet/protein.pdb
[the remark this answers is not in the document]
Comment (MCompChem) [reply industry_benchmarks/input_structures/prepared_structures/merck/cmet/protein.pdb:1]: Added the reason in the preparation details

File: industry_benchmarks/input_structures/prepared_structures/merck/cmet/PREPARATION_DETAILS.md (hunk 1 above)
Comment (MCompChem) [reply industry_benchmarks/input_structures/prepared_structures/merck/cmet/PREPARATION_DETAILS.md]: Added the version number in the preparation details

**Review by IAlibay — APPROVED** [review]
Looks great, thanks @MCompChem !

**Pull request merged**